dissect/target/helpers/localeutil.py

```diff
@@ -0,0 +1,52 @@
+import defusedxml.ElementTree as ET
```

Comment:
isort please :)

(We'll soon be adding this to the linting step)

Reply:
Fixed in [52c17e1](https://github.com/fox-it/dissect.target/pull/150/commits/52c17e14fc77910c72a9c428701f5810710d28e6).